cmake/scripts/common/HandleDepends.cmake

```diff
@@ -78,8 +78,10 @@ function(add_addon_depends addon searchpath)
           # make sure we create strings, not lists
           set(TMP_C_FLAGS "${CMAKE_C_FLAGS} ${ARCH_DEFINES}")
           set(TMP_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ARCH_DEFINES}")
+          set(TMP_EXE_LINKER_FLAGS "-L${OUTPUT_DIR}/lib ${CMAKE_EXE_LINKER_FLAGS} ${ARCH_DEFINES}")
```

Comment:
Why are define flags needed in the linker flags?

Reply:
Make no sense, was only a nightly c&p from above and my Mr. Brainbrain was on other side of world.

jenkins build this with addons please